### ISSUE TITLE
Integrate witn Accessibility Zoom

### ISSUE THREAD

**Issue by jecolon** [Caldis/Mos#431]
**这个问题是否源自某个缺陷？如果是，可于此处描述一下 | Is your feature request related to a problem? Please describe.**
I have a visual disability and use the macOS screen zoom feature all the time. I zoom in and out using the scroll wheel. It would be really helpful if that scroll to zoom was also smooth like when using an Apple mouse of trackpad.
**说说你的新点子 | Describe the solution you'd like**
Zooming in and out with the mouse scroll wheel would be smooth, just like scrolling a Web page.
**还有别的替代方案吗？ | Describe alternatives you've considered**
I've tried adjusting the scroll speed and step in both Mos and macOS System Preferences but they don't provide smooth zomm in / out.
**额外说明 | Additional context**
I'm using a non-apple mouse. Mos is working perfectly for all other scrolling operations. Just the zoom in and out seem to be missing.

**Comment by lawand** [comment]
This is needed because also the reverse scroll direction doesn't work with Accessibility Zoom at the moment.
